lxc/config.go

```diff
@@ -56,7 +101,12 @@ func (c *configCmd) run(config *lxd.Config, args []string) error {
 			return err
 		}
 
-		return fmt.Errorf(gettext.Gettext("Only 'password' can be set currently"))
+		if len(args) < 3 {
+			return fmt.Errorf(gettext.Gettext("Bad config command"))
```

Comment:
Can we return `errArgs` in all these cases? That way lxc will print out the help() for the command when someone does something wrong.